cmd/devnettest/requests/utils.go

```diff
@@ -25,3 +25,9 @@ func parseResponse(resp interface{}) (string, error) {
 
 	return string(result), nil
 }
+
+// GetNamespaceFromMethod splits a parent method into namespace and the actual method
+func GetNamespaceFromMethod(method string) (string, string) {
+	parts := strings.SplitN(method, "_", 2)
+	return parts[0], parts[1]
```

Comment:
This isn't safe ... should you get a string like "abcd" (i.e. eithout the underscore) the split function will return only a 1 element array hence calling parts[1] would cause undefined behavior.
Also the function name is "GetNameSpaceFromMethod" which makes me think only one value should be returned.